securedrop_client/app.py

```diff
@@ -150,7 +156,14 @@ def start_app(args, qt_args) -> None:
 
 
 def run() -> None:
+    config_file = "/etc/securdrop-client/client.ini"
     args, qt_args = arg_parser().parse_known_args()
+    if args.sdc_home == DEFAULT_SDC_HOME and \
+            os.path.exists(config_file):  # pragma: no cover
+        config = configparser.ConfigParser()
+        config.read(config_file)
+        args.sdc_home = config["client"]["homedir"]
```

Comment:
Why are you overriding the `sdc_home` after reading a config from `sdc_home`?

Reply:
I am reading the config from `/etc/securdrop-client/client.ini` file.

Reply:
Ok I still think this make no sense. The client is passed a `sdc_home`, then we read another value, then we sed that other value to `sdc_home`. I feel like this isn't a good pattern and is kind confusing.